_includes/nav_urls.html

```diff
@@ -0,0 +1,51 @@
+{% comment %}
+Get the appropriate URLs for the previous and next episodes,
+if they exist, based on whether the episode order is defined
+in site.contents or determined by numbered filename prefix.
+
+End result is two variables: prev_url and next_url.
+{% endcomment %}
```

Comment:
Could you please add description of `page.slug` here?  Perhaps, with a link to https://jekyllrb.com/docs/permalinks/ ?

Reply:
Thanks for the advice. I've added comment notes on this to `_includes/nav_urls.html` and `bin/boilerplate/_config.yml`